runner.py

```diff
@@ -1,5 +1,6 @@
 import sys
-sys.path.insert(0, './src/main/python/')
```

Comment:
This change only causes runner to fail loading

Reply:
I'm not sure why he made the change, but I fixed it in a way that should be robust for everybody.